plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -237,8 +237,12 @@ class Conan(
             return
         }
 
+        parseConanRemoteList(remoteList.stdout)
+    }
+
+    private fun parseConanRemoteList(remoteList: String) {
         // Iterate over configured remotes.
-        remoteList.stdout.lines().forEach { line ->
+        remoteList.lines().forEach { line ->
```

Comment:
Is `logger.warn { "Failed to list remotes." }` actually reachable? e.g. does `run()` internally call `requireSuccess()` which implies that `isError == false`?

Reply:
> Is `logger.warn { "Failed to list remotes." }` actually reachable?

Good point, it's not. As that issue was present before, I'll fix it in another commit.